VIDEO_5_SCRIPT.md

```diff
@@ -0,0 +1,44 @@
+# Video 5: Open Governance
+- What is open Governance
+- Why AsyncAPI is committed to this model
+- AsyncAPI and Linux Foundation
+
+Hello everyone, and thank you for joining us in this video. Today, we’re going to talk about Open Governance and why AsyncAPI is committed to this model. 
+
+Let's get started!
+
+## What is Open Governance
+
+Open Governance is a model of management based on transparency, collaboration, and ethics, which results in clearer accountability and management driven by the participation of all the people that make up a project, company, or institution, thus promoting a democratic system with inclusive growth.
+
+Open source has become an essential model for the success of open government initiatives. Both share, essentially, many of the same premises. Developers share information, ideas, and code to create more innovative software solutions, collectively and individually. This type of scalable and flexible software allows anyone with the source code to modify, improve, and redistribute it for greater accessibility and reuse. Open-source software works with the underlying principles of peer-to-peer production and mass collaboration, creating a more sustainable software development process for end-users. 
+
+The sharing of ideas and community-developed software has driven creative, scientific, and technological advancement in many industries: education, government, law, health, technology...This movement creates an opportunity for community members to collaborate, share, build and help each other to achieve common goals, in this case, through source code.
+
+## Why AsyncAPI is committed to this model
+
+AsyncAPI is committed to democratic management based on two vertebral aes: horizontality in decision making and transparency in project management. From the beginning of the specification, AsyncAPI is determined to rely on collective effort and group identity. The project focus on a governance and decision-making system in which values come first. The means are as necessary as the goal. Responsibilities are shared, understanding that success can only be enjoyed jointly.
```

Comment:
vertebral 'aes': I'm not sure the use of 'aes' fits this context.

Reply:
Do you know why the file appears different from the text extract of these comments? The line numbering and the text do not match.
<img width="1039" alt="Captura de Pantalla 2022-06-08 a las 16 42 02" src="https://user-images.githubusercontent.com/77982319/172645374-70f22950-f6a2-4c59-a5f3-198dde109906.png">

Reply:
oh, I see

Reply:
(https://user-images.githubusercontent.com/19964402/161353808-430cb73e-e2a3-4913-ad5e-6cccdeeaa997.png)

here's what  displays on my end though

Reply:
Does anyone know what could be happening? @derberg @alequetzalli

Reply:
oh, what is `aes` anyway 😄 

`## Why AsyncAPI is committed to this model` is not even in the file, hard to say what happened really, maybe some old version was commented, from a commit link, not directly in PR, like for example on https://github.com/asyncapi/training/pull/12/commits/924807e02fbc221b1c4e7b7650c32f96a35bd48f level 🤷🏼 hard to say really

Reply:
`aes `I meant to say "axes" but I had a typo.

`## Why AsyncAPI is committed` to this model is not even in the file, hard to say what happened really, maybe some old version was commented, from a commit link, not directly in PR, like for example on https://github.com/asyncapi/training/commit/924807e02fbc221b1c4e7b7650c32f96a35bd48f level 🤷🏼 hard to say really

Ok, thanks @derberg.